files/build_templates/docker_image_ctl.j2

```diff
@@ -7,18 +7,32 @@ HWSKU=`sonic-cfggen -m /etc/sonic/minigraph.xml -v minigraph_hwsku`
 start() {
     docker inspect --type container {{docker_container_name}} &>/dev/null
     if [ "$?" -eq "0" ]; then
+{%- if docker_container_name == "database" %}
+        echo "Starting existing {{docker_container_name}} docker with HWSKU $HWSKU"
         docker start {{docker_container_name}}
-    else
+        exit 0
+{%- else %}
+        dkrSku=`docker inspect --type container {{docker_container_name}} | python -c "import sys, json, os; print os.path.basename([x for x in json.load(sys.stdin)[0]['Mounts'] if x['Destination'] == '/usr/share/sonic/hwsku'][0]['Source'])"`
```

Comment:
>docker inspect --type container {{docker_container_name}} [](start = 16, length = 57)

can you combine this with line 8 so that you do not need to inspect twice?

Reply:
That makes sense. I've changed the code so there is only one docker inspect call.

They needs to be two statements, because the outer if only needs to test the result from docker inspection result, it doesn't need to see the python error if there is any.

Regards,
Ying